packages/hash/design-system/src/ui/loading-spinner.tsx

```diff
@@ -1,5 +1,5 @@
 import { Box, CircularProgress, circularProgressClasses } from "@mui/material";
-import { VFC } from "react";
+import React, { VFC } from "react";
```

Comment:
Let’s remove all default `React` imports for future proofing:
- https://twitter.com/dan_abramov/status/1308739731551858689
- https://epicreact.dev/importing-react-through-the-ages/

I personally prefer `import * as React from "react"`, but I think we use named imports in this codebase, so let’s do that consistently 🙂

Reply:
I had to explicitly import React because the JSX transform got applied in the `frontend` code but wasn't getting applied in a different setup like the Github PR block. Figured it worked in the `frontend` codebase since the version of Nextjs there supports that. I'll update to use the namespace import approach

Reply:
Just to confirm, I only made the updates to files in `design-system/*`. Also the major reason I made it was because I was initially getting the error about "React" not being in scope when trying to use any of the design system components in the Github PR Block. Didn't get the error when using the same components in the `frontend` code. I assume that's because our frontend/nextjs setup provides the JSX transform plugin that makes React available, but that isn't the case in other environments like the Github PR block

Reply:
Yup, just confirmed that's the cause